lib/slax/release.ex

```diff
@@ -23,7 +23,7 @@ defmodule Slax.Release do
   end
 
   defp load_app do
-    Application.ensure_started(:ssl)
+    Application.ensure_all_started(:ssl)
```

Comment:
wow was this little change the cause of the ssl issue?

Reply:
We'll find out in a few minutes 😅

Reply:
And actually part of the issue that Jason and I found is that the DB being used is the same as Slack-Agile so it was having trouble with the new migration I created